### ISSUE TITLE
Bedrock Ores compatibility?

### ISSUE THREAD

**Issue by Peri-42** [Kotori316/QuarryPlus#80]
In 1.12, there's a mod called Bedrock Ores, which adds super dense ores to the bedrock of a world, ores that you can mine a hundred times before they're replaced by bedrock, either by hand or with a special machine that does it for you, to get hundreds of ores.

However, whenever I run the QuarryPlus, it simply harvests these ores as if they were a single block, and you don't get any of the extra resources. I think this is because the block hardness is the same as other ores, they're just set to not go away until you've mined them a few hundred times to get all the resources out. The Quarry doesn't know this though, and just breaks the blocks to give a single ore.

This also leads to a situation where these blocks are supposed to turn into bedrock when you're done mining them out, but with the quarry you simply remove the block entirely, essentially removing bedrock.

Is there any way these blocks could be entirely skipped over by the miner, like bedrock is, so we can come back later and get these ores the way it's intended? Or perhaps to add a blacklist to the configs, so that we can add something like "bedrockores:*" and tell the mod to skip these blocks?

Thanks!

**Comment by Kotori316** [comment]
I added 2 options.

1. Black List for mining machines.
2. Support of "Bedrock Ores"

I'll write wiki article about black list. Here I tell you an example to add bedrock ore to the list.
Black List is loaded from a json located in `config/quarryplus/blacklist.json`. Update the mod and once you run quarry, the file will be generated.
In the json, replace all with below example.
```json
[
  {
    "id": "quarryplus:blacklist_air"
  },
  {
    "id": "quarryplus:blacklist_modid",
    "modID": "bedrockores"
  }
]
```

Support of "Bedrock Ores" is simply, quarry can get resources, exactly the same amount you can see as the popup of Bedrock Ore block. It works with `Bedrock Ores-MC1.12-1.2.7.42.jar`.

Thank you for your suggestion.

Edit:
I forgot to paste link to new version.
https://www.curseforge.com/minecraft/mc-mods/additional-enchanted-miner/files/2905166

**Comment by Peri-42** [comment]
Thanks so much for the quick response, this is awesome!

I did some testing with just a couple mods installed, and almost everything seems to work well. The blacklist works perfectly, I tried it out with both bedrock ores and some of the forestry beehives and it skipped right over them.

The Bedrock Ores compatibility mostly works, but I noticed that if I power the quarry with a low level of power (such as a few peat engines from forestry, or a single generator from draconic evolution), it only gets partway through mining the ore and then freezes completely. If I power it with a high level of power (such as with a creative RF source from draconic evolution), it works perfectly with no problems.

Thanks again for the work you've done!
